usaspending_api/api_docs/api_documentation/award_spending/recipient.md

```diff
@@ -1,3 +1,110 @@
-This is placeholder documentation for the /api/v2/award_spending/recipient
-endpoint.  Return all award spending by recipient for a given fiscal year
-and agency id.
+# State Data
+
+**Route:** /api/v2/award_spending/recipient/
+
+**Method:** GET
+
+This route returns basic information about the specified state.
+
+`:awarding_agency_id` fips code representing the state
+`:fiscal_year` year to filter on 
+`:limit` page length of results (*optional*)
+`:page` page to return (*optional*)
+
+## Response Example
+
+```
+{
+    "page_metadata": {
+        "count": 36608,
+        "page": 1,
+        "has_next_page": true,
+        "has_previous_page": false,
+        "next": "https://api.usaspending.gov/api/v2/award_spending/recipient/?awarding_agency_id=183&fiscal_year=2016&limit=10&page=2",
```

Comment:
@BrianZito shouldn't this just be 1

Reply:
When hitting the [endpoint directly](https://api.usaspending.gov/api/v2/award_spending/recipient/?fiscal_year=2016&awarding_agency_id=183), it returns a full url